lib/paperclip/color_extractor.rb

```diff
@@ -77,8 +78,8 @@ def make
     private
 
     def w3c_contrast(color1, color2)
-      luminance1 = (0.2126 * color1.r + 0.7152 * color1.g + 0.0722 * color1.b) + 0.05
-      luminance2 = (0.2126 * color2.r + 0.7152 * color2.g + 0.0722 * color2.b) + 0.05
+      luminance1 = color1.to_xyz.y * 0.01 + 0.05
```

Comment:
Okay, I seem to have misread [the formula](https://www.w3.org/TR/WCAG20-TECHS/G18.html#G18-tests), but I am also not familiar with what XYZ colorspace is and why you `0.01` it first.

Reply:
In the XYZ colorspace, Y is the luminance, expressed between 0 and 100

Reply:
I see. And it's already adjusted for the different colors? (The `0.7152` etc part)

Reply:
Y *is* the luminance, we don't use the other components at all.